### PR TITLE
Add support to projectd_65_ansi keyboard

### PR DESCRIPTION
**Pull request by gskygithub**
## Description

Add projectd_65_ansi keyboard

## QMK Pull Request 

https://github.com/qmk/qmk_firmware/pull/21520

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [ ] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

**Review by Xelus22 — CHANGES_REQUESTED** [review]
https://www.caniusevia.com/docs/layouts#kle-json-rules

Please colour in your JSON keys

File: v3/projectd/ansi/65/projectd_65_ansi.json [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+                ["RAINBOW_MOVING_CHEVRON", 15],
+                ["CYCLE_OUT_IN", 16],
+                ["CYCLE_OUT_IN_DUAL", 17],
+                ["CYCLE_PINWHEEL", 18],
+                ["CYCLE_SPIRAL", 19],
+                ["DUAL_BEACON", 20],
+                ["RAINBOW_BEACON", 21],
+                ["RAINDROPS", 23],
+                ["JELLYBEAN_RAINDROPS", 24],
+                ["HUE_BREATHING", 25]
```
[the remark this answers is not in the document]
Comment (gskygithub) [reply v3/projectd/ansi/65/projectd_65_ansi.json:48]: @Xelus22 The other effects are not applying to keyboard even though I added.

[the remark this answers is not in the document]
Comment (Xelus22) [reply v3/projectd/ansi/65/projectd_65_ansi.json:48]: You'll need to add them to this list. Or if you have enabled ALL effects, then you can use the default `"menus": ["qmk_rgb_matrix"],` and not have to use a custom one

example https://github.com/the-via/keyboards/blob/master/v3/xelus/pachi/xelus_pachi_rgb_rev1.json#L5-L6

**Comment by gskygithub** [comment]
@Xelus22 Resolved requested changes, thanks.

**Review by Xelus22 — APPROVED** [review]

**Pull request merged**